mindmaps-graql/src/test/java/io/mindmaps/graql/examples/ExamplesTest.java

```diff
@@ -37,7 +38,7 @@
 
     @Before
     public void setUp() {
-        MindmapsGraph graph = MindmapsTestGraphFactory.newEmptyGraph();
+        MindmapsGraph graph = Mindmaps.factory(Mindmaps.IN_MEMORY).getGraph(UUID.randomUUID().toString().replaceAll("-", "a"));
```

Comment:
The `UUID.randomUUID().toString().replaceAll("-", "a")` is just a lazy way of creating new graphs with unique names.